gef.py

```diff
@@ -6147,7 +6147,7 @@ def set_setting(self, argc, argv):
                 _newval = True if argv[1] in ("True", "TRUE", "true", "1") else False
             else:
                 _newval = argv[1]
-                _type( _newval )
```

Comment:
The point of this line is to trigger a Python type exception if the type is invalid. The affection is correctly done line 6156
``` 
__config__[ argv[0] ] = (_newval, _type)
```